src/widgets/overlay.ts

```diff
@@ -61,4 +69,9 @@ export default class AgsOverlay extends Gtk.Overlay {
         this._overlays.push(widget);
         super.add_overlay(widget);
     }
+
+    remove_overlay(widget: Gtk.Widget): void {
```

Comment:
remove_overlay is a gtk4 thing, not gtk3
you can probably do `remove(widget)` just like in box